Makefile

```diff
@@ -0,0 +1,16 @@
+IMAGE=wpengine/microservice
```

Comment:
nit and probably unlikely: For concurrent builds there's no guarantee you don't get someone elses image if you don't include branch_name/build_number the way -auth and -swagger did.